cardano-api/src/Cardano/Api/ProtocolParameters.hs

```diff
@@ -1638,3 +1588,45 @@ checkProtocolParameters sbe ProtocolParameters{..} =
      then return ()
      else Left . PParamsErrorMissingMinUTxoValue
                $ AnyCardanoEra era
+
+
+data ProtocolParametersError
+  = PParamsErrorMissingMinUTxoValue !AnyCardanoEra
+  | PParamsErrorMissingAlonzoProtocolParameter
+  deriving Show
+
+instance Error ProtocolParametersError where
+  displayError (PParamsErrorMissingMinUTxoValue (AnyCardanoEra era)) = mconcat
+    [ "The " <> show era <> " protocol parameters value is missing the following "
+    , "field: MinUTxoValue. Did you intend to use a " <> show era <> " protocol "
+    , " parameters value?"
```

Comment:
Are there golden tests for the render of this error?  I notice that there are two spaces between `protocol` and `value`.

Reply:
They're already present: https://github.com/input-output-hk/cardano-node/tree/d04f65532c80b431727c2db536adf2db9ae85f85/cardano-api/golden/files/errors/ProtocolParametersError